include/deal.II/lac/vector_internal.h

```diff
@@ -0,0 +1,1073 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+#ifndef dealii__vector_internal_h
```

Comment:
github displays that you still want to add `vector_internal.h`. Did you just forget to remove it when you renamed into `vector_operations_internal.h` or is this merely a visualization issue with github?